### PR TITLE
add blockadblock v3.2.1

### PR DESCRIPTION
**Pull request by maruilian11**
for #5153

### CODE REVIEW

File: ajax/libs/blockadblock/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sitexw/BlockAdBlock.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sitexw/BlockAdBlock.git",
+    "basePath": "",
+    "files": [
+      "blockadblock.js"
```
Comment (PeterDaveHello) [inline ajax/libs/blockadblock/package.json]: Why not `"blockadblock*.js"`?

File: ajax/libs/blockadblock/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "keywords": [
+    "adblock",
+    "advertising",
+    "ad",
+    "block",
+    "lock",
+    "detection",
+    "detecting"
+  ],
+  "homepage": "https://github.com/sitexw/BlockAdBlock",
```
Comment (PeterDaveHello) [inline ajax/libs/blockadblock/package.json]: Should be `http://fuckadblock.sitexw.fr/blockadblock/`

**Comment by PeterDaveHello** [comment]
Please use `npm` auto-update config.

**Comment by maruilian11** [comment]
**git repo url:** https://github.com/sitexw/BlockAdBlock
**Watch 15
  Star 187
 Fork 23**
@Amomo could u help me check this PR for #5137 ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

**Comment by PeterDaveHello** [comment]
Title...

**Pull request merged**